classes/class-sailthru-content.php

```diff
@@ -311,11 +311,17 @@ function generate_payload( $post, $post_id ) {
 		if ( has_post_thumbnail( $post->ID ) ) {
 			$image                          = wp_get_attachment_image_src( get_post_thumbnail_id( $post->ID ), 'full' );
 			$thumb                          = wp_get_attachment_image_src( get_post_thumbnail_id( $post->ID ), 'concierge-thumb' );
-			$post_image                     = $image[0];
-			$data['images']['full']['url']  = esc_attr( $post_image );
-			$post_thumbnail                 = $thumb[0];
-			$data['images']['thumb']['url'] = $post_thumbnail;
-		}
+                        $post_image                     = $image[0];
+                        $filename                       =  basename($post_image);
```

Comment:
Basename will get the file, but will also include any query params, see https://github.com/sailthru/sailthru-wordpress-plugin/issues/112